samples/peripheral/lpuart/README.rst

```diff
@@ -0,0 +1,55 @@
+.. _lpuart_sample:
+
+Low Power UART
+##############
+
+The Low Power UART sample demonstrates the capabilities of the :ref:`uart_nrf_lpuart` module.
+
+Overview
+********
+
+The sample implements a simple loopback using a single UART instance.
+The sample has console and logging disabled by default, to demonstrate low power consumption while having UART active.
+
+Requirements
+************
+
+* One of the following development boards:
+
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set18_start
+   :end-before: set18_end
```

Comment:
Shouldn't these two be `set22_start` and `set22_end`, respectively?

Reply:
well spotted. fixed